<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window1</class>
 <widget class="QMainWindow" name="window1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tyreButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
background-color: rgb(186, 23, 23);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tyres </string>
    </property>
   </widget>
   <widget class="QPushButton" name="steeringButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(186, 23, 23);</string>
    </property>
    <property name="text">
     <string>Steering</string>
    </property>
   </widget>
   <widget class="QPushButton" name="engineButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
background-color: rgb(186, 23, 23);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Engine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suspensionButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Segoe UI&quot;;
background-color: rgb(186, 23, 23);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Suspension</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email: </string>
    </property>
   </widget>
   <widget class="QLabel" name="contactLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contact us:</string>
    </property>
   </widget>
   <widget class="QLabel" name="phone_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Phone:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-91</x>
      <y>-21</y>
      <width>701</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 23, 23);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="main_header_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>531</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Gocars</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
